Assets/Scripts/Gameplay/Player.cs

```diff
@@ -6,13 +6,11 @@ namespace Gameplay
     [RequireComponent(typeof(Rigidbody2D))]
     public class Player : MonoBehaviour, IDamagable
     {
-        [SerializeField]
-        private float speed;
+        private float speed = 5f;
 
-        [SerializeField]
-        private float health;
+        private float health = 100f;
```

Comment:
`[SerializeField]` makes it so the values are editable in the editor

Reply:
Right I don't foresee that being something we're going to mess around with all the time, and anyone can add it back in as needed. Just simplifying things down

Reply:
Where do I go to disable the test?

Reply:
https://github.com/williamhammond/NotWorms/blob/main/Assets/Tests/Gameplay/PlayerTests.cs you can commit them out here.

I think speed at the least should stay configurable. Although i'd probably leave them all configurable for now. It'll make it easier to tweak the values while iterating. At least on my machine, even a small code change takes like a 10+ second assembly reload